### PR TITLE
Tweak README

### PR DESCRIPTION
**Pull request by kghbln**
* Update platform requirements. 1.39 is the first MW to support PHP 8 officially
* Proof-read the text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Parameters: none
 Platform requirements:
 
 * [PHP] 8.0 or later (tested up to 8.1)
-* [MediaWiki] 1.35 or later (tested up to 1.39 and master)
+* [MediaWiki] 1.39 or later (tested up to 1.41 and master)
```
Comment (malberts) [inline README.md:103]: We didn't drop 1.35 support: https://github.com/ProfessionalWiki/ExternalContent/blob/ea9185abdc2ad42faf267670d954afec3766e9ef/extension.json#L16

File: README.md [hunk 2]
```diff
@@ -247,6 +247,7 @@ Alternatively, you can execute commands from the MediaWiki root directory:
 ### Version 2.0.0 - 2023-10-30
 
 * Raised minimum PHP version from 7.4 to 8.0
+* Raised minimum MediaWiki version from 1.35 to 1.39
```
Comment (malberts) [inline README.md:250]: This did not change.

File: README.md (hunk 1 above)
Comment (kghbln) [reply README.md:103]: Nice that it is working. Officially MW 1.35 to 1.38 do not support PHP 8.0 which is what I wanted to convey here.

File: README.md (hunk 2 above)
Comment (kghbln) [reply README.md:250]: Admittedly, this is a mess, indeed.

**Comment by kghbln** [comment]
In the end, it does not matter.

**Pull request closed without merging**